### ISSUE TITLE
will the 29×1280×720 checkpoint will release

### ISSUE THREAD

**Issue by trouble-maker007** [PKU-YuanGroup/Open-Sora-Plan#358]

**Comment by LinB203** [comment]
Yes, we have open-sourced 29×720p and 93×480p models. Please note that the weights for 29×720p and 93×480p were trained on Panda70m and have not undergone final high-quality data fine-tuning, so they may produce watermarks.
Check the latest [resources](https://github.com/PKU-YuanGroup/Open-Sora-Plan?tab=readme-ov-file#-resource).

**Comment by trouble-maker007** [comment]
@LinB203 Thank you for your reply. I apologize for only noticing the checkpoint link in the report.
